checker/src/main/java/org/checkerframework/checker/determinism/DeterminismVisitor.java

```diff
@@ -525,6 +525,46 @@ public Void visitMethod(MethodTree node, Void p) {
         return super.visitMethod(node, p);
     }
 
+    /**
+     * If the declaration of {@code node} is annotated with {@code @RequiresDetToString}, checks
```

Comment:
The "nondeterministic.toString" error is only issued if the argument is `@Det`, but that isn't mentioned here.  So is the code wrong or the documentation?